apps/mobile-wallet/src/components/AddressesTokensList.tsx

```diff
@@ -121,30 +116,10 @@ const AddressesTokensListStyled = styled.View`
   position: relative;
 `
 
-const LoadingRow = styled.View`
-  flex-direction: row;
-  gap: 15px;
-  align-items: flex-start;
-  padding-top: 15px;
-  margin: 0 ${DEFAULT_MARGIN}px;
-`
-
-const LoadingOverlay = styled.View`
-  position: absolute;
-  top: 0;
-  bottom: 0;
-  left: 0;
-  right: 0;
-  background-color: ${({ theme }) => theme.bg.back1};
-  opacity: 0.8;
-`
-
-const Loader = styled.View`
-  position: absolute;
-  top: 30px;
-  left: 0;
-  right: 0;
+const LoadingSectionContainer = styled.View`
+  width: 100%;
   align-items: center;
+  padding-top: 100px;
```

Comment:
The spinner seems a bit cut-off at the bottom. I think adding a `padding-bottom: 40px` makes it look better.

```suggestion
  padding: 60px 0 40px;
```

<img width="350" src="https://github.com/user-attachments/assets/21be7ca3-361f-4b31-96a1-0365d51dc7e8" />